02-data-operations.Rmd

```diff
@@ -0,0 +1,228 @@
+
+# Data operations {#Data-operations}  
+
+## Common terminologies  
+
+The `mia` package uses classes defined by [`TreeSummarizedExperiment`](https://www.bioconductor.org/packages/release/bioc/html/TreeSummarizedExperiment.html), [`MicrobiomeExperiment`](https://github.com/FelixErnst/MicrobiomeExperiment) and in general [`SummarizedExperiment`](https://bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html).  
```

Comment:
This part would benefit greatly from a description of the class structure per se.

So maybe instead of putting the `mia` package front and center, put the `SummarizedExperiment`, ``SingleCellExperiment` and `TreeSummarizedExperiment` packages and the bioconductor ecosystem front and center

Reply:
Do you mean more textual information describing `SE` and explain BioC ecosystem?

Reply:
If I paraphrase this part it boils down to something like: "mia uses these classes, so that it".

The whole story is more like "There are some classes, which are used to contain rectangular count data, and can be used for microbiome analysis. ...." and the much further down something like "mia implements tools for microbiome analysis based on the classes described above".

That you are not experienced with these is not problem. Maybe read the first parts of SE, SCE and TSE vignettes to get a feeling for it and then rewrite this particular introduction.

Reply:
Okay, this should be possible.